patches/minecraft/net/minecraft/item/BowItem.java.patch

```diff
@@ -26,18 +32,38 @@
                    abstractarrowentity.func_234612_a_(playerentity, playerentity.field_70125_A, playerentity.field_70177_z, 0.0F, f * 3.0F, 1.0F);
                    if (f == 1.0F) {
                       abstractarrowentity.func_70243_d(true);
-@@ -100,6 +105,10 @@
+@@ -67,10 +73,13 @@
+ 
+                p_77615_2_.func_184148_a((PlayerEntity)null, playerentity.func_226277_ct_(), playerentity.func_226278_cu_(), playerentity.func_226281_cx_(), SoundEvents.field_187737_v, SoundCategory.PLAYERS, 1.0F, 1.0F / (field_77697_d.nextFloat() * 0.4F + 1.2F) + f * 0.5F);
+                if (!flag1 && !playerentity.field_71075_bZ.field_75098_d) {
+-                  itemstack.func_190918_g(1);
+-                  if (itemstack.func_190926_b()) {
+-                     playerentity.field_71071_by.func_184437_d(itemstack);
+-                  }
++                   pair.getValue().accept(itemstack);
++                   if (false) {
++                      itemstack.func_190918_g(1);
++                   if (itemstack.func_190926_b()) {
++                      playerentity.field_71071_by.func_184437_d(itemstack);
++                   }
++                   }
```

Comment:
The `if (false)` hack is useless if it makes the patch larger. Just removing the code is the best bet here I think.

Reply:
This should be resolved now?